docs/faq.rst

```diff
@@ -14,6 +14,25 @@ to NaCl.
 
 If you prefer NaCl's design, we highly recommend `PyNaCl`_.
 
+Why use ``cryptography``?
+-------------------------
+
+If you've done cryptographic work in Python before you have likely encountered
+other libraries in Python such as *M2Crypto*, *PyCrypto*, or *PyOpenSSL*. In
+building ``cryptography`` we wanted to address a few issues we observed in the
+legacy libraries:
+
+* Lack of PyPy and Python 3 support.
+* Lack of maintenance.
+* Use of poor implementations of algorithms (i.e. ones with known side-channel
+  attacks).
+* Lack of high level APIs.
+* Absence of algorithms such as
+  :class:`AES-GCM <cryptography.hazmat.primitives.ciphers.modes.GCM>` and
+  :class:`~cryptography.hazmat.primitives.kdf.hkdf.HKDF`.
+* Poor introspectability and thus poor testability.
+* Extremely error prone APIs and bad defaults.
```

Comment:
Can we reorder these to reflect the things we really think are most important these days? e.g. "Error prone APIs" should go at the top. Bad defaults should become "insecure defaults"